### PR TITLE
Fix model loading in datum explain

### LINKED ISSUE #721: explain command cannot find the model
When I try to infer an image with a model included in the project using the command 'explain', I get the error : 
**AttributeError: 'Tree' object has no attribute 'models'**

However, when I check the project info I see that the model is successfully added to the project, and the model can also produce annotations with 'model run' command 

Any help will be appreciated.
Screenshot from 2022-07-28 11-34-46

### PR DESCRIPTION
**Pull request by vinnamkim**
### Summary
 - Resolve #721
 - Fix model loading in `datum explain`
 - Add an integration test for `datum explain`

### How to test
This change will be covered in the added test.

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [x] I have added tests to cover my changes
- [x] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: datumaro/cli/commands/explain.py [hunk 1]
```diff
@@ -188,7 +188,13 @@ def explain_command(args):
 
     project = scope_add(load_project(args.project_dir))
 
-    model = project.working_tree.models.make_executable_model(args.model)
+    if args.model not in project.models:
+        raise CliException(
+            f"{args.model} has not benn registered in this project. "
```
Comment (bonhunko) [inline datumaro/cli/commands/explain.py]: typo: benn -> been

**Review by bonhunko — DISMISSED** [review]
looks good! I left a comment only for a typo thing.

**Review by sooahleex — DISMISSED** [review]
Looks good to me.

File: datumaro/cli/commands/explain.py (hunk 1 above)
Comment (vinnamkim) [reply datumaro/cli/commands/explain.py]: Fixed

**Review by bonhunko — APPROVED** [review]

**Pull request merged**